tests/unit/fem/test_linearform_ext.cpp

```diff
@@ -67,12 +69,23 @@ struct LinearFormExtTest
    LinearForm lf_dev, lf_std;
    LinearFormIntegrator *lfi_dev, *lfi_std;
 
+   static int GetQOrder(QuadratureType qtype, int p)
+   {
+      switch (qtype)
+      {
+         case Lobatto: return 2*p - 1;
+         case Legendre: return 2*p + 3;
+         case LegendreUnderIntegration: return 2*p -1;
```

Comment:
`2*p - 1`

Reply:
Fixed in 3581e69476c3338ed28115420a3b088fc4bea278